Add support for metadata in CBS (#4581)
kyma-project/kyma: docs/headless-cms/06-02-clusterdocstopic.md

| **spec.sources.type** |    **YES**   | Specifies the type of assets included in the ClusterDocsTopic CR. |
| **spec.sources.name** |    **YES**   | Defines a unique identifier of a given asset. It must be unique if there is more than one asset of a given type in a ClusterDocsTopic CR. |
| **spec.sources.mode** |    **YES**   | Specifies if the asset consists of one file or a set of compressed files in the ZIP or TAR format. Use `single` for one file and `package` for a set of files.  |

> | **spec.sources.metadata** |    **NO**   | Specifies the set of the metadata for ClusterAsset. It must be defined in a valid YAML/JSON format. |

Reviewer: ```suggestion
| **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the ClusterAsset. Define it in a valid YAML or JSON format. |
```
